jokers/rocket.lua

```diff
@@ -0,0 +1,245 @@
+    },
+    no_collection = true,
+    in_pool = function(self, args)
+        return false
+    end,
+    calculate = function(self, card, context)
+        if context.joker_main then
+            return {
+                x_mult = card.ability.extra.mult,
+                func = function()
+                    G.E_MANAGER:add_event(Event({
+                        trigger = "after",
+                        func = function()
+                            play_random_alien(0.75)
+                            return true
+                        end
+                    }))
+                end
+            }
+        end
+    end,
+    add_to_deck = function(self, card, from_debuff)
+        if not from_debuff then
+            local mult = folly_utils.pseudorandom_range(card.ability.extra.low, card.ability.extra.high, self.key)
+            card.ability.extra.mult = mult
+        end
+    end,
+    loc_vars = function(self, info_queue, card)
+        return { vars = { card.ability.extra.mult } }
+    end
+})
+
+SMODS.Joker({
+    key = "glorp_alien",
+    rarity = 4,
+    atlas = "folly_aliens",
+    pos = { x = 0, y = 0 },
+    soul_pos = { x = 4, y = 0 },
+    config = {
+        extra = {
+            low = 0,
+            high = 0.5,
+            x_mult = 1,
+        },
+    },
+    no_collection = true,
+    in_pool = function(self, args)
+        return false
+    end,
+    calculate = function(self, card, context)
+        if context.joker_main then
+            return {
+                x_mult = card.ability.extra.x_mult,
+                func = function()
+                    G.E_MANAGER:add_event(Event({
+                        trigger = "after",
+                        func = function()
+                            play_random_alien(1.5)
+                            return true
+                        end
+                    }))
+                end
+            }
+        end
+
+        if not context.blueprint and context.using_consumeable and context.consumeable.ability.name == "c_folly_strange_planet" then
+            card.ability.extra.x_mult = card.ability.extra.x_mult + folly_utils.pseudorandom_range(card.ability.extra.low, card.ability.extra.high, self.key)
+            return {
+                message = localize('k_folly_glorp')
+            }
+        end
+    end,
+    loc_vars = function(self, info_queue, card)
+        return {
+            vars = {
+                card.ability.extra.low,
+                card.ability.extra.high,
+                card.ability.extra.x_mult
+            }
+        }
+    end
+})
+
+SMODS.Consumable({
+    key = "strange_planet",
+    atlas = "folly_consumables",
+    pos = { x = 0, y = 0 },
+    set = "Planet",
+    --no_collection = true,
+    in_pool = function(self, args)
+        if next(SMODS.find_card("j_folly_rocket")) then
+            -- I could technically return the result from the next but it feels weird
+            return true
+        end
+        return false
```

Comment:
```suggestion
        return next(SMODS.find_card("j_folly_rocket")) ~= nil
```

we can check if its nil then and return a bool that way